example/src/index.jsx

```diff
@@ -0,0 +1,42 @@
+import 'core-js/stable';
+import 'regenerator-runtime/runtime';
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {
+  AppProvider,
+  ErrorPage,
+  PageRoute,
+} from '@edx/frontend-platform/react';
+import { APP_INIT_ERROR, APP_READY, initialize } from '@edx/frontend-platform';
+import { subscribe } from '@edx/frontend-platform/pubSub';
+import { messages as paragonMessages } from '@edx/paragon';
+import MyComponent from './MyComponent';
+
+import './index.scss';
+import '../../paragon/build/variables.css';
```

Comment:
Imports the output `variables.css` from building the theme in this repo with `style-dictionary` (i.e., with `build-tokens.js`). In reality, consuming MFEs could host that imported CSS file somewhere (e.g., on a CDN) such that it receives updates when the theme changes, pulling in the newly computed CSS variables, but without requiring a rebuild on consuming MFEs.